site/components/Form/InputField/index.tsx

```diff
@@ -9,13 +9,22 @@ interface Props extends InputHTMLAttributes<HTMLInputElement> {
   hideLabel?: boolean;
   errors?: { message?: string };
   errorMessageMap: (id: string) => string;
+  className?: string;
```

Comment:
Bit of a tricky one 🤔 
If we think about how we're using the `className` prop here, its purpose is to override the components default styling. Maybe the better thing to do is to rename the prop to something more appropriate but the intention is the same at the end of the day.

I don't think we should explicitly define `inputProps` in the type interface though, because that would mean we'll need to pass in the html attributes like (using the example in this PR):
```js
<InputField
  label={t({
    id: "resources.form.input.search.label",
    message: "Search",
  })}
  inputProps={{
    type: "search",
    autoComplete: "off"
    // etc
  }}
/>
```

Reply:
This will change all inputs on Pledge.
I opened an extra PR for that => https://github.com/KlimaDAO/klimadao/pull/677

Reply:
@0xAeterno 
Dang, saw your comment after I already opened the other PR 😄 

Let's move this discussion to there 👉 #677